### PR TITLE
Improvements object panel

### PR DESCRIPTION
**Pull request by ClementPasteau**
Note: "Edit extension options" should probably appear above the functions as it has nothing to do with functions.

image
image
image
image

### CODE REVIEW

**Comment by LuniMoon** [comment]
Tiny UI optim: the primary buttons that say "add..." need their "+" icon to the left.

Thank you for fixing the primary button to the bottom to keep it visible. 🙏

File: newIDE/app/src/ObjectsList/index.js [hunk 1]
```diff
@@ -827,9 +839,6 @@ const ObjectsList = React.forwardRef<Props, ObjectsListInterface>(
                     onEditItem={objectWithContext =>
                       onEditObject(objectWithContext.object)
                     }
-                    onAddNewItem={onAddNewObject}
```
Comment (4ian) [inline newIDE/app/src/ObjectsList/index.js:830]: This had a "smart" behavior: when the width was too small, the button would collapse by not showing its label anymore, but just the + icon. Something important to keep for small screens I think?

Comment (ClementPasteau) [reply newIDE/app/src/ObjectsList/index.js:830]: Yeah I noticed that, we can keep that indeed

**Comment by ClementPasteau** [comment]
> Totally. I think this should be moved into this icon, made permanent. This icon is for now only shown when you open events of a function/behavior/object: image

Done

**Review by 4ian — COMMENTED** [review]
A few things:
- All he `ResponsiveWindowMeasurer` everywhere would almost allow for the creation of a "ResponsiveRaisedButton"
- Are there lists in the app not following this new design?
- The "EditScene" icon is not really adapted to the extension, we should ask @LuniMoon for a new icon here.
- When you edit the name of an item, is the text field set with the proper size (same as the caption)?

**Review by 4ian — APPROVED** [review]
Lgtm!

**Pull request merged**